rasa_sdk/interfaces.py

```diff
@@ -271,6 +272,29 @@ def add_slots(self, slots: List[EventType]) -> None:
             self.slots[event["name"]] = event["value"]
             self.events.append(event)
 
+    def get_intent_if_not_fallback(self) -> Optional[Text]:
```

Comment:
How about something like this?
```suggestion
    def get_intent_of_last_message(self, skip_fallback_intent: bool = True) -> Optional[Text]:
```